### PR TITLE
Give descriptive name to link in Terms of Service

### LINKED ISSUE #3978: TOS "click here" link needs to be renamed
## Area or Component
Patient Experience - Terms of Service

## Quick Summary
One of the links in the terms of service is labelled simply as "click here", but this doesn't give enough information to users about what the link is.

## Accessibility Violation
The element's accessible name is missing or is not appropriate for the element's purpose. https://axe.deque.com/rule-help/en/aria-name-missing-incorrect 

See also the [links section of the Deque course.](https://dequeuniversity.com/class/semantic-structure/links/link-text#links-meaningful-text-alone:~:text=End%20code.-,The%20purpose%20of%20each%20link%20SHOULD%20be%20able%20to%20be%20determined%20from%20the%20link%20text%20alone.,-Screen%20reader%20users)

## Action Requested
Change the link text to:

> If you would like more information about the application of the Privacy Act at CDC, visit the [Health and Human Services website](https://www.hhs.gov/foia/privacy/index.html).

Spanish translation:
> Si desea obtener más información sobre la aplicación de la Ley de Privacidad en los CDC, visite el [sitio web del Departamento de Salud y Servicios Humanos](https://www.hhs.gov/foia/privacy/index.html).

Note: This change was approved by the CDC’s Office of General Counsel through Mike Judd. 

## Source Code (optional)
https://github.com/CDCgov/prime-simplereport/blob/c3e6b2dc330021e425e01657eaee613c51f9e11c/frontend/src/patientApp/timeOfTest/TermsOfService.tsx#L58-L58

## Original Deque Issue
aria-name-missing-incorrect | The element's accessible name is missing or is not appropriate for the element's purpose. | "click here" is not an accessible enough name; should describe where the link goes | Name: The element's name is missing or incorrect | https://axe.deque.com/rule-help/en/aria-name-missing-incorrect | critical | Yes | No | Interactive Elements | ["body > div#root:nth-of-type(1) > div.bg-base-lightest.minh-viewport:nth-of-type(1) > div.bg-base-lightest:nth-of-type(3) > main.patient-app.padding-bottom-4.bg-base-lightest.padding-top-05:nth-of-type(1) > form.grid-container.maxw-tablet.usa-prose:nth-of-type(1) > button#tos-consent-button.usa-button:nth-of-type(1)"] | The element's accessible name is missing or is not appropriate for the element's purpose. | <button type="button" class="usa-button" id="tos-consent-button" data-axe-pro-tab-stop-index="8">I agree</button> | wcag2a,wcag412b | rnv7@cdc.gov | Patient Self Reg - Terms of Service | https://stg.simplereport.gov/app/register/9C7BB |   | 2022-07-13T21:04:27.207Z
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --

### PR DESCRIPTION
**Pull request by mehansen**
# FRONTEND PULL REQUEST
## Related Issue
Resolves #3978 

## Changes Proposed
Update link destination and copy as per @lauraykerr's comments on the ticket

## Testing
Please review the screenshots

## Screenshots / Demos
Screen Shot 2022-10-17 at 11 08 43 AM
Screen Shot 2022-10-17 at 11 09 49 AM

### Accessibility
- [x] All changes have run through the Deque automated testing

### Content
- [x] Any content changes have been approved by content team

### CODE REVIEW

File: frontend/src/patientApp/timeOfTest/ToS.tsx [hunk 1]
```diff
@@ -59,8 +59,8 @@ function ToS() {
         i18nKey="testResult.tos.document.privacy.p0"
         parent="p"
         components={[
-          <a href="https://www.cms.gov/Research-Statistics-Data-and-Systems/Computer-Data-and-Systems/Privacy/PrivacyActof1974.html">
-            click here
+          <a href="https://www.hhs.gov/foia/privacy/index.html">
+            this content is replaced
```
Comment (mehansen) [inline frontend/src/patientApp/timeOfTest/ToS.tsx:63]: not sure if we have a preference for what to put here
the inner contents of the component [gets replaced](https://github.com/i18next/react-i18next/issues/1090#issuecomment-615276404) but the linter won't let me leave it empty

**Review by lauraykerr — APPROVED** [review]
LGTM, thanks!

**Review by zdeveloper — APPROVED** [review]
LGTM! I see it was already like that, but do you know reason why `privacy.p0` is broken up to many string in the English version and its only one string in the Spanish version?

**Comment by mehansen** [comment]
> LGTM! I see it was already like that, but do you know reason why `privacy.p0` is broken up to many string in the English version and its only one string in the Spanish version?

nope, I don't think there's a reason for either one. I was wondering if we had something that automatically broke up strings in our pre-commit hooks but doesn't look like it

**Pull request merged**